cosim/black-parrot-example/Makefile.design

```diff
@@ -57,6 +57,8 @@ $(CURR_VSRC_DIR)/bsg_bootrom.v: $(CURR_VSRC_DIR)/bsg_bootrom.rom
 
 BUILD_COLLATERAL  = $(CURR_VSRC_DIR)/bsg_blackparrot_pkg.sv
 BUILD_COLLATERAL += $(CURR_VSRC_DIR)/bsg_bootrom.v
+BUILD_COLLATERAL += $(CURR_VSRC_DIR)/bsg_bootrom.rom
+BUILD_COLLATERAL += $(CURR_VSRC_DIR)/bsg_bootrom.bin
```

Comment:
I think these should be an implicit dependency of bsg_bootrom.v?

Reply:
yes, but they need to be explicitly added to BUILD_COLLATERAL so they get removed on "make clean"

Reply:
nice catch!